src/lib/holocene/radio-input/radio-input.stories.svelte

```diff
@@ -22,35 +22,55 @@
   }}
 />
 
-<Story name="radio input">
+<Story
+  name="radio input"
+  args={{
+    disabled: false,
+  }}
+  template="radio-input"
+/>
+
+<Template id="radio-input" let:args>
```

Comment:
Just curious, what does Template do? TIA

Reply:
The template pattern is part of Storybook Args, which allows stories to get dynamic data via input arguments. So in this case we can toggle `disabled` and see the UI change in the story.